beacon_node/beacon_processor/src/scheduler/work_reprocessing_queue.rs

```diff
@@ -749,16 +752,24 @@ impl<S: SlotClock> ReprocessQueue<S> {
                 }
             }
             InboundEvent::Msg(DelayColumnReconstruction(request)) => {
+                let mut reconstruction_delay = QUEUED_RECONSTRUCTION_DELAY;
+                if let Some(seconds_from_current_slot) =
+                    self.slot_clock.seconds_from_current_slot_start()
+                    && seconds_from_current_slot >= RECONSTRUCTION_DEADLINE
```

Comment:
This logic may not work if we're triggering reconstruction on blocks other than the current slot. 
Could potentially be triggered here:
https://github.com/sigp/lighthouse/blob/f19d4f6af124994678a11a765e50dc4be7ddac34/beacon_node/network/src/network_beacon_processor/sync_methods.rs#L403-L417